packages/graph/hash_graph/lib/graph/src/store/postgres/query/compile.rs

```diff
@@ -321,6 +321,67 @@ impl<'c, 'p: 'c, T: PostgresQueryRecord + 'static> SelectCompiler<'c, 'p, T> {
         }
     }
 
+    fn add_special_relation_conditions(
+        &mut self,
+        relation: Relation,
+        current_alias: Alias,
+        current_table: AliasedTable,
```

Comment:
Is this distinction required anymore? Wouldn't `current_table` have an `alias` we can extract because of the other PR that merged? Or could the alias be different from the table? :)

Reply:
The aliases are different. I renamed the parameters to reflect this: c9acada553c58a03a7f13a69dccccd5bc8282f76